doc/cli/npm-outdated.md

```diff
@@ -3,7 +3,7 @@ npm-outdated(1) -- Check for outdated packages
 
 ## SYNOPSIS
 
-    npm outdated [<name> [<name> ...]]
+    npm outdated [[@<scope>/]<pkg> ...]
```

Comment:
Same point about `<@scope>` – I think this one is probably on me originally, in that I've changed my mind about what makes the most sense. So I can make this change if it seems too exasperating to you.